paper-toolbar.html

```diff
@@ -227,6 +229,37 @@
         width: auto;
         margin: 0;
       }
+
+      /* TODO(noms): Until we have a better solution for classes that don't use
+       * /deep/ create our own.
+       */
+      .layout.horizontal {
+        @apply(--layout-horizontal);
+      }
+
+      .layout.center {
+        @apply(--layout-center);
+      }
+
+      .layout.start-justified {
+        @apply(--layout-start-justified);
+      }
+
+      .layout.center-justified {
+        @apply(--layout-center-justified);
+      }
+
+      .layout.end-justified {
+        @apply(--layout-end-justified);
+      }
+
+      .layout.around-justified {
+        @apply(--layout-around-justified);
+      }
+
+      .layout.justified {
+        @apply(--layout-justified);
+      }
```

Comment:
I think so -- that's what the `cumputeBarClassName` does, if I understand correctly. You can specify what the justification and it gets applied to the top/middle etc. So you still need to know what the class "end-justified" is